scheduling/pulse.py

```diff
@@ -0,0 +1,15 @@
+import os
+from pyspark import SparkConf, SparkContext
+from pyspark.sql import SQLContext
+from python_etl.testpilot import pulse
+
+if __name__ == "__main__":
+    conf = SparkConf().setAppName('python_etl')
+    sc = SparkContext(conf=conf)
+    sqlContext = SQLContext(sc)
+
+    # Make bdist_egg available to executer nodes
+    sc.addPyFile(os.path.join('dist', os.listdir('dist')[0]))
+
+    # Run job
+    pulse.etl_job(sc, sqlContext)
```

Comment:
How does the date get calculated for this job? Many airflow jobs take a date specified by the scheduler via environment variable ([$date as instantiated by`ds_nodash`](https://github.com/mozilla/telemetry-airflow/blob/master/dags/main_summary.py#L24)). The scheduler is free to handle retries and backfill that don't necessarily match the current time, resulting in generally consistent results.

If you want to add this to the cli, Click has a neat feature to take an argument from the environment: `@click.argument('date', envvar='date')`.